code/src/styling/media-queries.css

```diff
@@ -0,0 +1,243 @@
+
+
+@media print {
+    body {
+  color: yellow;
+  background-color: greenyellow;
+    }
+  
+  }
+  
```

Comment:
hmm i never seen this '@media print' syntax, so im not completely sure what this is doing

Reply:
It's styling not for a specific browser width but rather for how the page should look like when printing on paper. (So for a recipe for instance you might disable the navbar and ads and only print the recipe). But this i never finished.